src/garage/torch/q_functions/discrete_cnn_q_function.py

```diff
@@ -110,3 +111,21 @@ def __init__(self,
                          output_b_init=output_b_init,
                          layer_normalization=layer_normalization,
                          is_image=is_image)
+
+    # pylint: disable=arguments-differ
+    def forward(self, observations):
+        """Return Q-value(s).
+
+        Args:
+            observations (np.ndarray): observations of shape `input_shape`.
```

Comment:
where can the user reference `input_shape`?

Reply:
Hmm you're right, fixed.